modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7513,6 +7513,8 @@ feature.flag.LPD-31212.description=Users can create rich text content that inclu
 feature.flag.LPD-31212.title=Disable XSS Validations in Rich Text Editor
 feature.flag.LPD-31789.description=Allow administrators to configure Google Vertex AI as text embedding provider for semantic search.
 feature.flag.LPD-31789.title=Google Vertex AI Text Embedding Provider
+feature.flag.LPD-34650.description=Developers can apply theme CSS and global CSS client extensions to an entire instance.
+feature.flag.LPD-34650.title=Instance Theme CSS and Global CSS Client Extensions
```

Comment:
On the frontend we always refer to the globalCSS client extension as just "CSS Client Extension".

Reply:
Same for the description

Reply:
Ah good, missed that. Thanks.